tutor/src/components/tabs.cjsx

```diff
@@ -75,15 +75,12 @@ Tabs = React.createClass
   renderTab: (tab, index) ->
     isSelected = index is @state.activeIndex
 
-    <li key={index} tabIndex={index} className={classnames(active: isSelected)}>
-      <a role="tab"
-        href="#"
-          tabIndex={if isSelected then -1 else 0}
-        onClick={_.partial(@onTabClick, index)}
+    <li key={index} tabIndex={if isSelected then -1 else 0} className={classnames(active: isSelected)} role="tab">
+      <h2 onClick={_.partial(@onTabClick, index)}
         aria-selected={'true' if isSelected}
       >
         {tab}
-      </a>
+      </h2>
     </li>
 
   render: ->
```

Comment:
ok, in that case the only sticking point is whether it's a good idea to have `h2` be a clickable element accessibility wise.  What's the rational for changing them?  I know h2 has better SEO, but we're not worried about that here 😁